alpine/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:latest
```

Comment:
You'll want to specify an explicit Alpine version here (like `alpine:3.6`, for example) so that when 3.7 is released, it doesn't inadvertently break your build. :+1: